app/src/main/java/com/nervousfish/nervousfish/activities/SettingsActivity.java

```diff
@@ -289,4 +285,72 @@ public boolean onOptionsItemSelected(final MenuItem item) {
             return super.onOptionsItemSelected(item);
         }
     }
+
+    private static final class BindPreferenceToValueListener implements Preference.OnPreferenceChangeListener {
+        @Override
+        public boolean onPreferenceChange(final Preference preference, final Object newValue) {
```

Comment:
Don't we need to validate here. If we do however look below this code, there are more examples of this